internal/conf/conf.go

```diff
@@ -61,6 +61,7 @@ type EnvironmentConfig interface {
 	Config
```

Comment:
I'm OK with the way it is now but I'm also thinking, would it be simpler if you put Strip onto Config instead of EnvironmentConfig?

That would save you from having to cast the interface to EnvironmentConfig later.  It may also keep things simpler from the documentation & user's point of view.  There's no obvious reason why, as a user, I should not be allowed to put 'strip' at the top-level of the config and am forced to put it under a 'prefix'.  I might want to do that if all of my prefixes have a common leading component (it is somewhat realistic, e.g. imagine we're trying to enable the tool for multiple subtrees on CDN, all of which will need the same "uploadhost:/67570/rcm" part stripped).

Reply:
> That would save you from having to cast the interface to EnvironmentConfig later.
+1
That would resolve the Strip() concern for me as well